modules/structs/repo_actions.go

```diff
@@ -32,3 +32,37 @@ type ActionTaskResponse struct {
 	Entries    []*ActionTask `json:"workflow_runs"`
 	TotalCount int64         `json:"total_count"`
 }
+
+// CreateActionWorkflowDispatch represents the payload for triggering a workflow dispatch event
+// swagger:model
+type CreateActionWorkflowDispatch struct {
+	// required: true
+	// example: refs/heads/main
+	Ref string `json:"ref" binding:"Required"`
+	// required: false
+	Inputs map[string]any `json:"inputs,omitempty"`
```

Comment:
It's better to use https://gitea.com/gitea/act/src/branch/main/pkg/jobparser/model.go#L175-L181

Reply:
It seems to me that this section is about the input schema in the actions YAML file. This doesn't fit here because this part contains the input values provided by the user. For example:

```json
{
   "ref": "refs/heads/main",
   "inputs": {
        "value1": "my input",
        "value2": false
    }
}
```

Reply:
The handling of the input values is done by a function, [see below](https://github.com/go-gitea/gitea/pull/32059/files/0357003ddf2e25f48538b91f0b8e2dcf6045fd5d#diff-14c22a886c54df9b8950ae2f32a3524d61b10e9be0754c4a43513895f5da9cf8R259-R279).